lib/index.d.ts

```diff
@@ -1108,11 +1108,15 @@ declare namespace Stardog {
         }
 
         type AllVgOptions = SharedOptions & RdbmsOptions & MongoOptions & CsvOptions;
-
+        
```

Comment:
Some formatting isn't right here (extra spaces, and 4 spaces vs. 2).

Reply:
oops, think i never changed my defaults for mac, and got into an overreliance on Prettier.